ansible/roles/media-metadata-sync/templates/media-metadata-sync-cronjob-configmap.yml.j2

```
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: {{ NAMESPACE }}
+  name: media-metadata-sync-cronjob-configmap
+  labels:
+    app: media-metadata-sync-cronjob
+data:
+  NODE_OPTIONS: "--max-old-space-size=3072"
```

to high, if the pod has 2048 MB of Memory then an old space size form 3072 MB is to big. 
Set it to an value like 1536 MB.
```suggestion
  NODE_OPTIONS: "--max-old-space-size=1536"
```